build.gradle

```diff
@@ -141,6 +141,7 @@ subprojects {
     version = '5.2.3-SNAPSHOT'
     ext {
         bouncyCastleVersion = '1.70'
+        commonsLang3Version = '3.12.0'
```

Comment:
This appears in another PR you added and again, this does not seem important enough to add another dependency.

Reply:
Or you can just copy the functions needed, means isBlank and isNotBlank, from commons-lang3, which is an apache v2 licensed repo from apache, so get some codes from there seems resonable, as this is alsy an apache v2 licensed repo from apache

Reply:
POI has its own StringUtil class - you can add extra methods there.